Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -141,6 +141,41 @@ namespace AzToolsFramework::Prefab
             FocusOnPrefabInstanceOwningEntityId(entityId);
         }
 
+        // Set the selection to the entities owned by the newly focused prefab instance.
+        // Only do it if the focused prefab is not the root.
+        if (auto focusedInstance = GetInstanceReference(m_rootAliasFocusPath);
+            focusedInstance.has_value() && focusedInstance->get().GetParentInstance() != AZStd::nullopt)
```

Comment:
nit: looks like you can call `focusedInstance->get().HasParentInstance()` directly